routers/web/repo/actions/actions.go

```diff
@@ -168,7 +168,7 @@ func List(ctx *context.Context) {
 	actionsConfig := ctx.Repo.Repository.MustGetUnit(ctx, unit.TypeActions).ActionsConfig()
 	ctx.Data["ActionsConfig"] = actionsConfig
 
-	if len(workflowID) > 0 && ctx.Repo.IsAdmin() {
+	if len(workflowID) > 0 && ctx.Repo.CanWrite(unit.TypeActions) {
 		ctx.Data["AllowDisableOrEnableWorkflow"] = true
```

Comment:
I agree with the idea that users with write access can manually run workflows. But I think only repo admins should be able to disable or enable workflows.